src/gossip/node.zig

```diff
@@ -51,11 +51,13 @@ pub const ContactInfo = struct {
     version: Version,
     addrs: ArrayList(IpAddr),
     sockets: ArrayList(SocketEntry),
+    extensions: ArrayList(void) = ArrayList(void).init(undefined),
```

Comment:
would probably be better to have the same empty enum in rust client so its more clear

Reply:
Sounds good. In my mind, an empty enum or struct is basically a newtype for void, since they are both unit types. But I can see void being confusing since it doesn't explain anything to a reader.

The only problem is getting this to compile. `const Extension = union(enum) {};` triggers a compilation error that I can't make sense of and may be a compiler bug. `const Extension = enum {};` and `const Extension = union {};` seem to reveal limitations in the serialization logic. With `const Extension = void;` it compiles no problem, and I can add a doc explaining its purpose.